source/server/setup/setup-service-configurations.js

```diff
@@ -0,0 +1,23 @@
+let env = Space.getenv.multi({
+  loginStyle: ['SPACE_ACCOUNTS_SERVICE_CONFIGURATION_LOGIN_STYLE', 'popup', 'string'],
+  googleClientId: ['SPACE_ACCOUNTS_SERVICE_CONFIGURATION_GOOGLE_CLIENT_ID', '', 'string'],
+  googleSecret: ['SPACE_ACCOUNTS_SERVICE_CONFIGURATION_GOOGLE_SECRET', '', 'string']
+});
+
+let setupGoogleServiceConfiguration = function() {
+
+  if (env.googleClientId !== '' && env.googleSecret !== '') {
```

Comment:
@rhyslbw, @DominikGuzei: What do you think about updating services configuration for production environment. I think I should limit this to dev only or update this code to do only insert for production environment. Or only do insert for  prod environment and log to error if config needs to be updated (new configuration is provided by environment variables that is different from current one in the db). 

I should probably switch to insert and update with checking and not just do upsert every time when environment variables exist.

Environment variables are very specific though...